spec/lib/listen/adapter/polling_spec.rb

```diff
@@ -61,7 +61,6 @@
         t = Thread.new { subject.start }
         sleep 0.25
         t.kill
-        t.join
```

Comment:
Do you need to call `join` after `kill`?

Reply:
No, `kill` is synchronous, so it won't return until the thread is killed. `join` is unnecessary at that point.